Helper/ConfigHelper.php

```diff
@@ -1198,17 +1198,6 @@ public function setCustomerGroupsEnabled(bool $val, ?string $scope = null, ?int
         );
     }
 
-    /**
-     * @param $storeId
-     * @return bool
-     */
-    public function credentialsAreConfigured($storeId = null)
-    {
-        return $this->getApplicationID($storeId) &&
-            $this->getAPIKey($storeId) &&
-            $this->getSearchOnlyAPIKey($storeId);
-    }
-
```

Comment:
Can we keep this method and mark as deprecated and then remove it after EOL of this release?

Reply:
Sure, I'll re-add it :)